zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/tenant/TenantRemoveEntityProcessor.java

```diff
@@ -79,7 +83,7 @@ final var record = command.getValue();
       return;
     }
 
-    if (!validateEntityAssignment(command)) {
+    if (!validateEntityAssignment(command, tenantId)) {
```

Comment:
🔧 Why? The command already contains the tenant id, right? I don't see the need of this separate parameter. For all the methods you added the parameter actually 😄 

I guess it's a matter of preference, reducing parameters vs retrieving the data from the command multiple times. I don't really care strongly about it so it's whatever you want.

Reply:
I'll keep like this to be aligned with `TenantAddEntityProcessor`